R/layer.R

```diff
@@ -334,7 +334,8 @@ Layer <- ggproto("Layer", NULL,
         && length(aes_n) > 0 && all(aes_n == 1) && n > 1) {
       cli::cli_warn(c(
         "All aesthetics have length 1, but the data has {n} rows.",
-        i = "Did you mean to use {.fn annotate}?"
+        i = "Please consider using {.fn annotate} or providing this layer \\
+        with data with a single row."
```

Comment:
```suggestion
        i = "Please consider using {.fn annotate} or provide this layer \\
        with data containing a single row."
```

I think this reads better